1.1/jessie/build-projectjson/README.md

```diff
@@ -10,10 +10,10 @@ This repository contains images that are used to compile/publish ASP.NET Core ap
 ## Supported tags
 
 - [`1.1.0-projectjson`, `1.1-projectjson`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/build-projectjson/Dockerfile)
```

Comment:
FYI - there is a proposal to move `latest` to msbuild.  We are moving forward with this in [microsoft/dotnet](https://github.com/dotnet/dotnet-docker/pull/200)